BitmapUtil/BitmapUtil.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Drawing.Imaging;
```

Comment:
It looks like this change from FreeImage to Pfim is breaking the displaying of the map (wrong references for the images)

It now looks like this
![image](https://github.com/ZeromusXYZ/AAEmu-Tools/assets/8280756/5982403a-f05c-4bbe-b7d9-cf029db5991c)

Instead of this
![image](https://github.com/ZeromusXYZ/AAEmu-Tools/assets/8280756/85715580-73ce-466e-8188-a06e20420fd9)

Reply:
The [sample code in Pfim's readme](https://github.com/nickbabcock/Pfim#usage) tries to warn of such case and suggests calling `var handle = GCHandle.Alloc(image.Data, GCHandleType.Pinned);` to preserve the image buffer returned by Pfim from being Garbage Collected and reused. Sadly this also means that if you don't want to leak memory, the handle needs to be freed when Bitmap is no longer in use.

My thought was that Bitmap constructors without explicit input data would be memory managed by the Bitmap class itself (hence the reason why a cloned copy of the instance is returned), but turns out then that this is quite not the case. I will take a look at it again to find an alternative solution.

Reply:
Please take a look at it again, I changed it to create a new Bitmap and then have the graphics API to draw the original Bitmap onto the new one. This way the resulting Bitmap should be completely independent from the source.